composer.json

```diff
@@ -34,6 +34,7 @@
 		"test": "phpunit"
 	},
 	"require-dev": {
-		"scrutinizer/ocular": "^1.3"
+		"scrutinizer/ocular": "^1.3",
+		"phpunit/phpunit": "^5.2"
```

Comment:
No, but we should :-) My opinion: we will not get stable until EOL of 5.5
So I am :+1: for php5.6 as min